includes/Core/Util/Migration_1_46_0.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Migration for 1.46.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Authentication\Google_Proxy;
+use Google\Site_Kit\Core\Authentication\Profile;
+use Google\Site_Kit\Core\Authentication\Verification_File;
+use Google\Site_Kit\Core\Authentication\Verification_Meta;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
+use WP_User;
+use WP_Error;
+
+/**
+ * Class Migration_1_46_0
+ *
+ * @since 1.46.0
+ * @access private
+ * @ignore
+ */
+class Migration_1_46_0 {
```

Comment:
I have been testing this on my end and it seems like we don't need this migration class at all. The tooltip tour won't re-appear after reset because dismissed feature tours are saved in the persistent option. Thus I think we can remove this class.

Reply:
cc @felixarntz for visibility.

Reply:
@eugene-manuilov yes, in that scenario it would be fine, but we're trying to avoid a different one:

https://github.com/google/site-kit-wp/issues/4133#issuecomment-958647430

Reply:
@eugene-manuilov @ivankruchkoff is correct - this extra flag is needed to indicate that the plugin was already connected before. In combination with the check for whether the module is not manually activated, this tells us that:
* the site was using Site Kit before and was already connected
* the site didn't have PSI connected

Because of those conditions we need to show the tooltip tour. On reset, this option here will get wiped, which is intentional because after reset the site starts in a fresh state. In that situation, we shouldn't ever show the tooltip tour because activating PSI after setup is more of an expected scenario, while if the site already _was_ connected, it would be a more "random" experience where we have to explain why it's suddenly active.

Reply:
Thanks, @felixarntz and @ivankruchkoff.